src/Compilers/Core/Portable/Emit/EditAndContinue/DeltaMetadataWriter.cs

```diff
@@ -645,6 +669,15 @@ protected override void CreateIndicesForNonTypeMembers(ITypeDefinition typeDef)
             implementingMethods.Free();
         }
 
+        private bool DefinitionExistsInAnyPreviousGeneration(IDefinition item) => item switch
```

Comment:
>DefinitionExistsInAnyPreviousGeneration [](http://example.com/codeflow?start=21&length=39)

It looks like each of the callers uses a single distinct case from the switch expression. Consider inlining the single case at each call-site and removing this method.